code/modules/fallout/reagents/drugs.dm

```diff
@@ -374,3 +374,72 @@
 	M.adjustOrganLoss(ORGAN_SLOT_HEART, 20)
 	..()
 	return
+
+/datum/reagent/drug/steady
+	name = "Steady Fluid"
+	description = "A potent, untested drug which increases the user's accuracy."
+	color = "#E0D532"
+	reagent_state = LIQUID
+	overdose_threshold = 20
+	addiction_threshold = 9 //Addicted on the first dose
+	metabolization_rate = 0.8 * REAGENTS_METABOLISM
+	ghoulfriendly = TRUE
+
+/datum/reagent/drug/steady/on_mob_add(mob/living/carbon/human/M)
+	..()
+	if(isliving(M))
```

Comment:
1. `mob/living/carbon/human/M` on L388 typecasts M, so that it'll only ever be `mob/living/carbon/human` (or a subtype of) or `null`.
2. `isliving()` checks if something is a `mob/living` or subtype.
3. The check `if(M)` will be sufficient to check if M exists _and_ is a `mob/living/carbon/human` (or, as you can guess, a subtype.)